package.json

```diff
@@ -42,6 +42,9 @@
   },
   "dependencies": {
     "classnames": "^2.2.6",
+    "markdown-it": "^10.0.0",
+    "markdown-it-highlightjs": "^3.0.0",
+    "mdi-react": "^6.7.0",
```

Comment:
If we are able to find replacements for:

```
src/components/NormalLeftRow/index.jsx:import WarningIcon from '@material-ui/icons/ReportProblemOutlined';
src/components/NormalRightRow/index.jsx:import InfoIcon from '@material-ui/icons/Info';
```

then we should be able to remove the `@material-ui/icons` dependency. mdi-react will typically have all of the material-ui icons.

Reply:
Applied changes!